controllers/comments-controller.js

```diff
@@ -9,4 +10,19 @@ exports.getCommentsByArticleId = (req, res, next) => {
     }).catch((err) => {
         next(err);
     });
+}
+
+exports.addCommentsByArticleId = (req, res, next) => {
+    const articleId = req.params.article_id;
+    const {body, username} = req.body;
+    return getArticleId(articleId)
```

Comment:
This would be a perfect time for a re usable check exists function and a promise.all. Check the advanced error handling.